ui/app/templates/topology.hbs

```diff
@@ -370,21 +370,29 @@
               {{else}}
                 <div class="columns is-flush">
                   <div class="dashboard-metric column">
-                    <p data-test-node-count class="metric">
+                    <p data-test-node-count class="metric topo">
                       {{this.model.nodes.length}}
                       <span class="metric-label">
                         Clients
                       </span>
                     </p>
                   </div>
                   <div class="dashboard-metric column">
-                    <p data-test-alloc-count class="metric">
+                    <p data-test-alloc-count class="metric topo">
                       {{this.scheduledAllocations.length}}
                       <span class="metric-label">
                         Allocations
                       </span>
                     </p>
                   </div>
+                  <div class="dashboard-metric column">
+                    <p data-test-node-pool-count class="metric topo">
+                      {{this.model.nodePools.length}}
```

Comment:
Trying to think about the `all` pool here as well, I'm not sure if I would expect to see it counted 🤔 

But it may be weird to _not_ count it when you try to compare it with something like the `nomad node pool list` output, so it's probably better to count it? 

I think I will ask the rest of the teams for seconds (and thirds, fourths....) opinions 😄